pkg/build/openbsd.go

```diff
@@ -10,10 +10,13 @@ import (
 	"strconv"
 	"time"
 
+	"github.com/google/syzkaller/pkg/log"
 	"github.com/google/syzkaller/pkg/osutil"
 )
 
-type openbsd struct{}
+type openbsd struct {
+	useGCE bool
```

Comment:
This is available in the `build` function as `vmType` argument, so no need to store it here.

Reply:
Done